### DISCUSSION TITLE
Unable to add rules in CRUD operations

### DISCUSSION THREAD

**Discussion by adnan-shaikhx** [mockoon/mockoon#1250] [Help]
Unable to find Add rules option for CRUD operation. Herein I'm attaching the screenshot 
![image](https://github.com/mockoon/mockoon/assets/80242963/328ad83a-bd11-47ff-8985-8869f98d8321)

**Comment by 255kb** [comment]
According to the [documentation](https://mockoon.com/docs/latest/api-endpoints/crud-routes/#crud-route-differences:~:text=Also%2C%20CRUD%20routes,else%20still%20applies.) the first response is fixed, linked to a databucket and always the default one. 
You can still add more responses and assign rules to them. They will be served if the rule is valid, otherwise, it will default to the CRUD operations. 

A typical setup could be: 
1. mandatory "CRUD operation" response
2. 401 Unauthorized response with a Authorization Header rule
3. ...
